packages/react-components/src/organisms/EventCard.tsx

```diff
@@ -1,19 +1,97 @@
 import React, { ComponentProps } from 'react';
 import { EventResponse } from '@asap-hub/model';
+import { isAfter, isBefore, subMinutes } from 'date-fns';
 
 import { ToastCard, TagList, EventInfo } from '../molecules';
+import { Link } from '../atoms';
+
+const EVENT_CONSIDERED_IN_PROGRESS_BEFORE_MINUTES: number = 5;
+
+const materials: Array<keyof EventCardProps> = [
+  'notes',
+  'videoRecording',
+  'presentation',
+  'meetingMaterials',
+];
 
 type EventCardProps = ComponentProps<typeof EventInfo> &
-  Pick<EventResponse, 'tags' | 'status'>;
-const EventCard: React.FC<EventCardProps> = ({ status, tags, ...props }) => (
-  <ToastCard
-    toastText={
-      status === 'Cancelled' ? 'This event has been cancelled' : undefined
+  Pick<
+    EventResponse,
+    | 'tags'
+    | 'status'
+    | 'meetingLink'
+    | 'notes'
+    | 'videoRecording'
+    | 'presentation'
+    | 'meetingMaterials'
+  >;
+const EventCard: React.FC<EventCardProps> = ({ status, tags, ...props }) => {
+  const toastCardProps = (): Omit<
+    ComponentProps<typeof ToastCard>,
+    'children'
+  > => {
+    const currentTime = new Date();
```

Comment:
Yeah you're gonna have to re-write this component to use the `useDateHasPassed` hook. `new Date()` in render is reading outside state and will not cause an update on its own